src/rust/src/merkle_frontier.rs

```diff
@@ -67,16 +69,16 @@ impl<H: Copy + Hashable + HashSer> MerkleFrontier<H> {
         root
     }
 
-    /// Returns the number of leaves appended to the frontier.
+    /// Returns the number of leaves in the Merkle tree for which this is the frontier.
```

Comment:
Is this a change in semantics?

Reply:
No, this has the same meaning, it's just clarifying (since the previous comment there didn't specify "appended to since when.")